src/app/services/locales/en-us/index.yaml

```diff
@@ -8,3 +8,6 @@ services:
         data_plane_proxies: 'Data Plane Proxies'
     items:
       title: Services
+  detail:
+    enabled: 'Enabled'
+    disabled: 'Disabled'
```

Comment:
I think these go away if you remove the TLS from externalServiceDetails

Reply:
Removed it.